libs/s25main/world/NodeMapBase.h

```diff
@@ -31,6 +31,7 @@ class NodeMapBase final : public MapBase
 
 public:
     using Node = T_Node;
+    typedef typename std::vector<Node>::const_iterator const_iterator;
```

Comment:
For consistency. Will apply this now as the rest looks ok.
```suggestion
    using const_iterator = typename std::vector<Node>::const_iterator;
```